### DISCUSSION TITLE
Give conditions color or a gray fill

### DISCUSSION THREAD

**Discussion by Recte** [home-assistant/frontend#7415] [Automations / scripts / scenes]
The functionality of the frontend/GUI is expanding rapidly and that is a great thing. One of the improvements is the ability do build complex and/or/not-conditions. It could just be me, but I find it hard sometimes to see the level I am at. Of course I could check the yaml, but the whole idea is to make the frontend as powerful/useful as possible. So I was thinking what you would think of adding some color or gray shading. I attached an example of both.

I personally believe using shades of color, linked to the condition type is the most useful option. The colors I choose are just illustrative by the way and i'd suggest using colors that are already used in the frontend. What do you think about this idea?

![Condition fill - color](https://user-images.githubusercontent.com/26491601/96670150-fc8ec280-135e-11eb-838a-86b7c0dc56af.png)

![Condition fill - gray](https://user-images.githubusercontent.com/26491601/96670148-fbf62c00-135e-11eb-8d3a-af6d0ff74cbb.png)

**Comment by spacegaier** [comment]
I am preparing currently a PR to add this. Looking for some feedback reg. the coloring: https://github.com/home-assistant/frontend/pull/7768

**Update**: PR closed based on feedback.

**Reply by Recte** [reply]
Out of curiosity, is there any update on making the cards comply to the Material Design like described in this PR  https://github.com/home-assistant/frontend/pull/7768#issuecomment-757048501 ?

Nowadays you can nest choices (choose) and be creative with the conditions within them, which makes it very hard to keep track the way it is displayed right now. I really think it would be received very well once the visualization is improved.
